README.rst

```diff
@@ -58,6 +58,26 @@ Usage
       deployment_option='Multi-AZ',
       region='us-east-1'
     ) # 0.35
+..
+
+**Note** : AWS provides the pricing list in json format with no warranty of format change.  If a format change by AWS causes a breaking change; the below mitigation can be used at the cost of "stale" pricing.
+
+.. code-block:: python
+
+    import awspricing
+
+    """Known EC2 versions - [20180108224013, 20171128215034, 20171026015458,
```

Comment:
I think providing the known EC2 versions here may harm more than help, as people are more likely to hardcode an out-of-date version. Perhaps we could provide a link to the URL which has the latest version instead?

Reply:
Yes I can do that. Or if non-current version is used I can print a warning; whatever your preference.

Reply:
I can see there being legitimate cases for using an older version (e.g processing billing data from the past), so not sure if a warning makes sense. What do you think about providing a simple bash script to grab the latest version at the time it's run?